### PR TITLE
Add default profile fallback image

### PR DESCRIPTION
**Pull request by TheGamerzs**

### CODE REVIEW

**Comment by TheGamerzs** [comment]
@Timeraa This is ready to push

File: .gitignore [hunk 1]
```diff
@@ -5,5 +5,5 @@ dist
 .vscode/settings.json
 src/langs
 .nuxt
-package-lock.json
-package-lock.json
+yarn.lock
+languageUpdater.js
```
Comment (ririxidev) [inline .gitignore:9]:
```suggestion
package-lock.json
languageUpdater.js
```

File: src/pages/users/_userId.vue [hunk 2]
```diff
@@ -254,26 +256,29 @@
...
+			user.name = !user.name.includes("Deleted User")
+				? user.name
+				: userPresences[0]?.author?.name ||
+				  userContributions[0]?.contributors.find(user => {
+						if (user.id === params.userId) return user;
+				  })?.name ||
+				  "Unknown user";
 
-			user.tag = user.tag || "????";
+			console.log(userPresences[0]?.author?.name);
```
Comment (ririxidev) [inline src/pages/users/_userId.vue:267]: console log, yes
```suggestion
```

**Comment by TheGamerzs** [comment]
@Timeraa

**Pull request merged**